bundles/core/pom.xml

```diff
@@ -495,6 +495,11 @@
             <version>3.0.0-magento232</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.intellij</groupId>
```

Comment:
Why do we need this dependency?

Reply:
This one is required for the `@NotNull` annotation. The same the WCM datasource servlets use as well. For reference, see https://github.com/adobe/aem-core-wcm-components/blob/development/bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/AllowedHeadingElementsDataSourceServlet.java#L31.

Reply:
I figured it out eventually, but we should use the same annotations from `javax.annotation` (e.g. `javax.annotation.Nonnull`). Introducing a dependency on a vendor library for some trivial annotations seems wrong to me.

Reply:
Done :)